Add Elastic Cloud to to SAASINTs Cloud Cost documentation (#24638)
DataDog/documentation: content/en/cloud_cost_management/saas_costs.md

[104]: https://docs.snowflake.com/en/user-guide/key-pair-auth#assign-the-public-key-to-a-snowflake-user

{{% /tab %}}

{{% tab "Elastic Cloud" %}}

> Your Elastic Cloud cost data for the past 15 months can be accessed in Cloud Cost Management after 24 hours. To access the available data collected by each SaaS Cost Integration, see the [Data Collected section](#data-collected).

Reviewer: ```suggestion
In Cloud Cost Management, you can access your Elastic Cloud cost data for the past 15 months after 24 hours. To access the available data collected by each SaaS Cost Integration, see the [Data Collected section](#data-collected).
```

> 1. In your browser, see the [API Key][102] section in your Elastic Cloud organization's settings.
> 2. Click the **Create New Key** button.
> 3. Choose a **Name** and **Expiration Date** for your API key.
> 4. Select the **Billing Admin** role.
> 5. Click the **Create Key** button to generate the key.
> 7. Navigate to the [Elastic Cloud integration tile][101] in Datadog and click **Add Account**.
> 7. Enter your **Elastic Cloud Organization ID** and **Billing API Key** in the account table on the Elastic Cloud Integration tile in Datadog.

Reviewer: ```suggestion
1. Go to the [API Key][102] section in your Elastic Cloud organization's settings.
2. Click **Create New Key**.
3. Choose a **Name** and **Expiration Date** for your API key.
4. Select the **Billing Admin** role.
5. Click **Create Key** to generate the key.
6. Go to the [Elastic Cloud integration tile][101] in Datadog
7. Click **Add Account**.
8. Enter your **Elastic Cloud Organization ID** and **Billing API Key** in the account table.
```

| `network_access_type` | Network access type for the cluster. Possible values are `INTERNET`, `TRANSIT_GATEWAY`, `PRIVATE_LINK`, and `PEERED_VPC`. |
| `product` | Product name. Possible values include `KAFKA`, `CONNECT`, `KSQL`, `AUDIT_LOG`, `STREAM_GOVERNANCE`, `CLUSTER_LINK`, `CUSTOM_CONNECT`, `FLINK`, `SUPPORT_CLOUD_BASIC`, `SUPPORT_CLOUD_DEVELOPER`, `SUPPORT_CLOUD_BUSINESS`, and `SUPPORT_CLOUD_PREMIER`. |

{{% /tab %}}
{{% tab "Elastic Cloud" %}}
| Tag Name | Tag Description |
|---|---
| `name` | The unique identifier of the Elastic Cloud resource. |
| `price_per_hour` | The cost of the Elastic Cloud resource per hour. |

> | `kind` | The kind of resource. |

Reviewer: ```suggestion
| `kind` | The type of resource. |
```
Is this still accurate? I think it's better to use another word in the description so the definition is less circular.
